lib/Site.js

```diff
@@ -378,14 +381,17 @@ Site.prototype.collectUserDefinedVariablesMap = function () {
       userDefinedVariables[id] = html;
     });
   });
+  logger.verbose('User-defined variables collected.');
 };
 
 Site.prototype.generate = function (baseUrl) {
   // Create the .tmp folder for storing intermediate results.
+  const startTime = new Date();
   logger.profile(GENERATE_SITE_LOGGING_KEY);
   fs.emptydirSync(this.tempPath);
   // Clean the output folder; create it if not exist.
   fs.emptydirSync(this.outputPath);
+  logger.info('Start generating website.');
```

Comment:
Use a timestamp for this, otherwise this message doesn't seem to be necessary because the user knows that he is building the website, that's why he ran `markbind build` or `markbind serve`. :/

```javascript
logger.info(`Website generation started at ${startTime.toLocaleTimeString()}`);
```